component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticatorConstants.java

```diff
@@ -24,28 +24,22 @@
  */
 public class EmailOTPAuthenticatorConstants {
 
-    private EmailOTPAuthenticatorConstants() {
-
-    }
-
     public static final String AUTHENTICATOR_NAME = "EmailOTP";
     public static final String AUTHENTICATOR_FRIENDLY_NAME = "Email OTP";
-
+    public static final String IS_IDENTIFIER_FIRST_INITIATED_FROM_AUTHENTICATOR = "isIdfInitiatedFromAuthenticator";
+    public static final String NOTIFY_USER_EXISTENCE = "Recovery.NotifyUserExistence";
```

Comment:
Without defining this constant here, can't we import the constant defined at the recovery component?